packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -158,7 +158,8 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
           id={ id }
           tabIndex={ disabled ? -1 : 0 }
           aria-expanded={ isOpen ? 'true' : 'false' }
-          role='menu'
+          role='combobox'
+          aria-readonly='true'
```

Comment:
this should not be aria-readonly="true" instead it should be aria-autocomplete="none"